src/components/Tooltip/index.tsx

```diff
@@ -63,16 +63,16 @@ function TooltipUI(props: TooltipUIProps): ReactElement {
                 ref={targetRef}
             >
                 {target}
-                {visible && (
```

Comment:
the onClick on the parent div was getting called when interacting with the tooltip content.